### PR TITLE
[vamp-sdk] install pkgconfig files

### LINKED ISSUE #22653: [vamp-sdk] pkgconfig files not installed
The vamp-sdk port uses its own CMake build system. There are pkgconfig files in the vamp-sdk sources but the CMakeLists.txt in the port does not configure or install these.

### PR DESCRIPTION
**Pull request by Adela0814**
Fix #22653

install pkgconfig file:
```
vamp.pc
vamp-hostsdk.pc
vamp-sdk.pc
```

### CODE REVIEW

File: ports/vamp-sdk/portfile.cmake [hunk 1]
```diff
@@ -5,14 +5,19 @@ vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
 )
 
+vcpkg_replace_string("${SOURCE_PATH}/pkgconfig/vamp.pc.in" "%PREFIX%" "@PREFIX@")
+vcpkg_replace_string("${SOURCE_PATH}/pkgconfig/vamp-hostsdk.pc.in" "%PREFIX%" "@PREFIX@")
+vcpkg_replace_string("${SOURCE_PATH}/pkgconfig/vamp-sdk.pc.in" "%PREFIX%" "@PREFIX@")
+
```
Comment (ras0219-msft) [inline ports/vamp-sdk/portfile.cmake:11]: I believe these should not be needed at all -- `vcpkg_fixup_pkgconfig()` will delete and recreate the `prefix=...` line anyway.

```suggestion
```

**Review by ras0219-msft — CHANGES_REQUESTED** [review]
Thanks for the PR!

File: ports/vamp-sdk/portfile.cmake (hunk 1 above)
Comment (Adela0814) [reply ports/vamp-sdk/portfile.cmake]: removed, thanks for your comment.

**Comment by dg0yt** [comment]
Does `vamp-hostsdk.pc` provide useful information given vcpkg's split of host triplet and target triplet?

**Comment by dg0yt** [comment]
> @dg0yt They are different libraries and has different sources.

Sure. But it doesn't invalidate my question. I could rephrase:
Does vamp-hostsdk.pc relate to tools or libs to be used on the host?

**Comment by JackBoosY** [comment]
@dg0yt Nope, their `add_library` are unconditional.

**Comment by Be-ing** [comment]
>  Does vamp-hostsdk.pc relate to tools or libs to be used on the host?

"Host" in this context means an application that hosts plugins. It is unrelated to vcpkg's term "host".

**Pull request merged**